### PR TITLE
Extend "draw only pins" to qfpchips and muxdemuxes

### PR DESCRIPTION
**Pull request by Rmano**
Applying @Skillmon 's #550 to `qfpchip`. 

Mux-demuxes to go...

### CODE REVIEW

**Comment by Rmano** [comment]
@Skillmon  I think it's done. A very useful addition, so thanks again...

File: doc/circuitikzmanual.tex [hunk 1]
```diff
@@ -6050,9 +6052,32 @@ \subsubsection{Mux-Demux customization}
 \texttt{logic ports draw input leads} (default \texttt{true}) or, locally,
 with the style \texttt{no inputs leads} (that can be reverted with
 \texttt{input leads}).
-
 The main difference between setting \texttt{external pins width} to \texttt{0} or using \texttt{no inputs lead} is that in the first case the normal pin anchors and the border anchors will coincide, and in the second case they will not move and stay where they should have been if the leads were drawn.
 
+You can draw only selected pins on the and leave out the rest by setting the keys
```
Comment (Skillmon) [inline doc/circuitikzmanual.tex:6057]: Typo after "pins on the".

File: doc/circuitikzmanual.tex [hunk 2]
```diff
@@ -6050,9 +6052,32 @@ \subsubsection{Mux-Demux customization}
...
+\texttt{multipoles/draw only \emph{side} pins} and the corresponding style
+\texttt{draw only \emph{side} pins} where \texttt{\emph{side}} can be \texttt{left}, \texttt{right},
+\texttt{top}, \texttt{bottom}.
+Those key accepts a comma separated list of
```
Comment (Skillmon) [inline doc/circuitikzmanual.tex:6061]: Should be "Those keys"

File: doc/circuitikzmanual.tex (hunk 1 above)
Comment (Rmano) [reply doc/circuitikzmanual.tex:6057]: Thanks, will fix in #552

**Pull request merged**